src/Core/Cache/MemcachedCacheDriver.php

```diff
@@ -56,9 +60,15 @@ public function __construct(array $memcached_hosts)
 	/**
 	 * (@inheritdoc)
 	 */
-	public function getAllKeys()
+	public function getAllKeys($prefix = null)
 	{
-		return $this->memcached->getAllKeys();
+		// Doesn't work because of https://github.com/php-memcached-dev/php-memcached/issues/367
```

Comment:
@MrPetovan did you get it for Memcached working? I disabled it because of the known issue

with:
memcached  1.4.25-2ubuntu1.4
php-memcached 2.2.0-51-ge573a6e+2.2.0-2build2

Reply:
It worked on my Debian RPS:
- memcached `1.4.33`
- php-memcached `3.0.4+2.2.0-3+0~20180910140656.5+stretch~1.gbpa130f3`

[The provided issue](https://github.com/php-memcached-dev/php-memcached/issues/367) mentions memcached 1.5+, maybe we should check Memcached version before throwing out the Exception?

Reply:
It's a little bit confusing with the bugs in the `getAllKeys()` method.
- One says it doesn't work with memcached 1.5+ [here](https://github.com/php-memcached-dev/php-memcached/issues/367
- One says it doesn't work with php-memcache before 3.0.1 [there](https://github.com/php-memcached-dev/php-memcached/issues/203#issuecomment-282041918)